server/views/pages/prisonerSummary/prisonerSummary.njk

```diff
@@ -0,0 +1,110 @@
+{% extends "partials/layout.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "macros/renderFields.njk" import renderFields %}
+
+{% set pageTitle = applicationName + " - Report" %}
+{% set bodyClasses = "app-view-report" %}
```

Comment:
we don’t need the particular changes this class does for this page, but i'm thinking… maybe it _is_ a nice idea to give all pages unique class names? that way, we can make unit/integration tests know for sure what template was rendered 🤔
or just remove this line for now 😆